server/server.js

```diff
@@ -205,6 +209,8 @@ UserPresence = {
 
 	start: function() {
 		Meteor.onConnection(function(connection) {
+            connection.closed = true;
```

Comment:
does it make sense to set a connection as `closed` right after it have been created?